tests/sync/test_purge_fuzz_attempts_command.py

```
@@ -0,0 +1,329 @@
+        FuzzAttempt.objects.filter(id=old_attempt.id).update(created_at=old_created)
+
+        out = StringIO()
+        call_command("purge_fuzz_attempts", "--days=5", stdout=out)
+
+        self.assertEqual(FuzzAttempt.objects.count(), 0)
+        self.assertIn("Successfully purged", out.getvalue())
+
+    def test_keeps_recent_records(self):
+        """Should keep records newer than retention days."""
+        # Create recent record
+        recent_attempt = FuzzAttempt.objects.create(
```

Ruff (0.15.1)

[error] Local variable `recent_attempt` is assigned to but never used

Remove assignment to unused variable `recent_attempt`

(F841)